pycbc/distributions/sky_location.py

```diff
@@ -31,5 +35,66 @@ class UniformSky(angular.UniformSolidAngle):
     _default_polar_angle = 'dec'
     _default_azimuthal_angle = 'ra'
 
+class FisherSky():
+    """A distribution that returns a random (ra, dec) angle drawn from the
+    Fisher distribution. Assume that the concentration parameter (kappa)
+    is large so that we can use a Rayleigh distribution about the north
+    pole and rotate it to be centered at the (ra, dec) coordinate mu.
+    Assume kappa = 1 / (0.66*sigma)**2
+    As in UniformSky, the declination (dec) varies from pi/2 to-pi/2
+    and right ascension (ra) varies from 0 to 2pi. The angles
+    should be provided in (ra,dec) format in radians (the variable
+    angle_unit='rad' by default), rather than factors of pi, or
+    in degrees (in this case set angle_unit='deg').
+    References:
+      * http://en.wikipedia.org/wiki/Von_Mises-Fisher_distribution
+      * http://arxiv.org/pdf/0902.0737v1 (states the Rayleigh limit)
+    """
+    name = 'fisher_sky'
+    _params = ['ra', 'dec']
+
+    def __init__(self, angle_unit='rad', **params):
+        self.sigma = params['sigma']
+        self.kappa = 1./((0.66*params['sigma']))**2
```

Comment:
Note that sigma must also be converted from deg to rad when `angle_unit == 'rad'`.

Reply:
Sigma conversion included